scripts/Get-Tags.ps1

```diff
@@ -0,0 +1,32 @@
+param(
+    [string]$Branch='master',
+    [string]$ImageBuilderImageName='microsoft/dotnet-buildtools-prereqs:image-builder-jessie-20171005132855',
+    [string]$RepoName
+)
+
+$ErrorActionPreference = 'Stop'
+$repoRoot = Split-Path -Path "$PSScriptRoot" -Parent
+
+if ([String]::IsNullOrWhiteSpace($RepoName))
+{
+    [Uri]$remoteUrl = New-Object -TypeName System.Uri -ArgumentList "https://github.com/dotnet/dotnet-docker-nightly.git"
```

Comment:
What is the point in initializing this when it is an out parameter?

Reply:
As discussed offline, this can be simplified as `$remoteUrl = $null`

---
In reply to: [143577592](https://github.com/dotnet/dotnet-docker-nightly/pull/455#discussion_r143577592) [](ancestors = 143577592)